packages/compass-connections/src/connections-manager.ts

```diff
@@ -243,6 +244,20 @@ export class ConnectionsManager extends EventEmitter {
         );
       });
 
+      dataService.on?.('oidcAuthFailed', (error) => {
+        this.logger.warn(
+          'ConnectionsManager',
+          mongoLogId(1_001_000_317),
+          'Failed to authenticate with OIDC',
+          error
+        );
```

Comment:
Fwiw, we should already be logging this from oidc-plugin in all cases

Reply:
yeah correct! we do that in the plugin. removing it now

Reply:
removed in [1bb4438](https://github.com/mongodb-js/compass/pull/5879/commits/1bb4438fcbcfa1cef64dfeb0fb4232d0b5f63ab5)